quinn-proto/src/congestion.rs

```diff
@@ -0,0 +1,27 @@
+//! Logic for controlling the rate at which data is sent
+
+use std::time::Instant;
+
+mod new_reno;
+pub use new_reno::{NewReno, NewRenoConfig};
+
+/// Common interface for different congestion controllers
+pub trait Controller: Send {
+    /// Packet deliveries were confirmed
+    fn on_ack(&mut self, sent: Instant, bytes: u64, congestion_blocked: bool);
+
+    /// Packets were deemed lost or marked congested
+    fn on_congestion_event(&mut self, now: Instant, sent: Instant, persistent: bool);
```

Comment:
The meaning of `persistent` is not obvious to me. Different name? Some comments? Switch from `bool` to `enum`?

Reply:
Elaborated a bit and expanded the name to the full term used in the recovery draft.